hathor/transaction/block.py

```diff
@@ -162,7 +162,7 @@ def get_funds_fields_from_struct(self, buf: bytes) -> bytes:
 
         return buf
 
-    def get_graph_fields_from_struct(self, buf: bytes) -> bytes:
+    def get_graph_fields_from_struct(self, buf: bytes, *, verbose: bool = False) -> bytes:
```

Comment:
Here also we should use `verbose` in `get_graph_fields_from_struct`, right?

And also log `self.data`.

Reply:
Fixed. Thanks!